src/collection-facets/more-facets-content.ts

```diff
@@ -470,18 +525,10 @@ export class MoreFacetsContent extends LitElement {
     return [
       srOnlyStyle,
       css`
-        @media (max-width: 560px) {
-          section#more-facets {
-            max-height: 450px;
-          }
-          .facets-content {
-            overflow-y: auto;
-            height: 300px;
-          }
-        }
```

Comment:
Moved these media queries down below, because otherwise the unconditional rules below them took precedence.